man/systemd.network.xml

```diff
@@ -1200,6 +1200,47 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
           </para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>IPv4NFTSet=</varname><replaceable>family</replaceable>:<replaceable>table</replaceable>:<replaceable>set</replaceable></term>
+        <term><varname>IPv6NFTSet=</varname><replaceable>family</replaceable>:<replaceable>table</replaceable>:<replaceable>set</replaceable></term>
```

Comment:
The is per-address setting. It is not necessary to introduce two settings. Please simply introduce `NFTSet=`.

Reply:
I think I thought that there could be several addresses and they could be either v4 or v6. But with only one address, one setting should work, awesome!